tests/runner.py

```diff
@@ -1194,6 +1194,13 @@ def show():
            % (expected))
   atexit.register(show)
 
+def suite_for_module(module):
+  import parallel_runner
```

Comment:
My general opinion on imports is that they should go together at the top, except maybe if one (an expensive one?) is unlikely to be used. Not sure what convention we use in emscripten though.